Notification.js

```diff
@@ -162,6 +170,10 @@ Notification.defaultProps = {
   backgroundColour: 'white',
   notificationBodyComponent: DefaultNotificationBody,
   iconApp: null,
+  onShowing: () => {},
+  onShown: () => {},
+  onClosing: () => {},
+  onClosed: (willShowAgain) => {}
```

Comment:
No need to define this variable here is there?

Reply:
I defined it to show function signature. I'm passing an argument to the function to inform if the notification will close and immediately open again.
I needed onClosed method to remove notification from being rendered. If user shows a new notification while it's already open, onClosed(true) will be fired, so I know that I should keep rendering the notification until onClosed(false) is called, otherwise the new notification won't be shown and I get an unhandled exception.

Reply:
I think this sort of information should go inside the docs that you're written for that prop instead of defining an unused variable, alternatively you could put a readable comment in?

Reply:
added to docs and removed unused argument